doc/integrator/backend_qgis.rst

```diff
@@ -22,8 +22,8 @@ In the ``cgxp_print`` plugin add the following configuration:
 
 .. code:: javascript
 
-   encodeLayer: { useNativeAngle: true, serverType: 'qgis' }
-   encodeExternalLayer: { useNativeAngle: true, serverType: 'qgis' }
+   encodeLayer: { useNativeAngle: true, serverType: 'QGISSERVER' }
+   encodeExternalLayer: { useNativeAngle: true, serverType: 'QGISSERVER' }
```

Comment:
The first attemps with 'qgis' throw an error message giving the value list available: MAPSERVER,GEOSERVER,QGISSERVER